### DISCUSSION TITLE
Enhancement: The abillity to "bookmark" websites within the CasaOS ui

### DISCUSSION THREAD

**Discussion by AlexFigas** [IceWhaleTech/CasaOS#314] [Ideas]
I'm constantly going to my Google Drive so I can check how are my uploads doing with rclone. 
I find it would be nice to have some kind of "app" that is a redirect to a website of our choice, just like a redirect to the docker image, but in this case a website.

![image](https://user-images.githubusercontent.com/65889776/174475676-088fbc9a-8100-4a5f-80d7-dd237ccd6f9c.png)

Something with these 3 options would be enough.

Thank you

**Comment by JohnGuan** [comment]
Related Issue #333
